assets/src/dashboard/components/typeaheadOptions/index.js

```diff
@@ -118,19 +126,63 @@ const TypeaheadOptions = ({ isOpen, items, maxItemsVisible = 5, onSelect }) => {
   }, [hoveredIndex, items, onSelect, isOpen]);
 
   useEffect(() => {
-    if (listRef.current) {
-      listRef.current.scrollToItem(0);
+    if (!listRef.current) {
+      return () => {};
```

Comment:
We can just return with no value since these functions in this file and the next do not have any cleanup.